dev_guide/builds.adoc

```diff
@@ -912,3 +912,131 @@ case `*scmsecret*`:
 <1> The URL of private repository is usually in the form
 `git@example.com:<username>/<repository>`.
 ====
+
+
+== Limiting the type of build strategies for builds
```

Comment:
since this requires cluster-admin, maybe it belongs under the admin guide, not the dev guide?